lib/player.js

```diff
@@ -2395,6 +2526,11 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
           fullyLoaded.resolve();
         });
 
+
+    // Check status of the LCEVC DIL Object and reset or
+    // create or close based on config
+    this.setupLcevc_(this.config_);
```

Comment:
You can't use LCEVC with src=.  Without MediaSource playback, we can't send the necessary information into the DIL.

Reply:
Removed this